manifests/client.pp

```diff
@@ -51,4 +55,6 @@
     Class['ssh::client::install']
     -> Class['ssh::client::config']
   }
+
+  create_resources('ssh::client::match_block', $match_block)
```

Comment:
Create resources is a function, which is marked as deprecated. The Splat operator is the replacement:


```
$match_block each |String $matchblock_name, Hash $matchblock_details| {
  ssh::client::match_block { $matchblock_name:
    * => $matchblock_details,
  }
}
```

Reply:
@saz seems likek, I am not able to find this too, but I found the note here: https://puppet.com/docs/puppet/7/function.html#create-resources

Reply:
I used `#create_resources` to be consistent with the existing code and because it is more concise.  @saz Do you want me to convert it to splat block?

Reply:
@jhoblitt I'm planning on changing this part. Working on some more updates of this module before the next release :smile:

Reply:
OK - it sounds like I'm not being asked to change this PR.